proposals/notifications.md

```diff
@@ -0,0 +1,169 @@
+single resource or span across different resources. A single resource with
+multiple notifications may be suitable for `HTTP PATCH`, whereas multiple
+resources containing a single notification may be suitable for `HTTP POST` or
+`PUT`. If different access controls are desired for inboxes and notifications,
+implementing a multiple notification approach with `HTTP POST` or `PUT` may be
+preferable.
+
+
+## Namespaces and document conventions
+The following namespaces are used in this document:
+
+|Prefix|Namespace
+|------|---------
+|solid|http://www.w3.org/ns/solid/terms#
+|xsd|http://www.w3.org/2001/XMLSchema#
+|dcterms|http://purl.org/dc/terms/
+|sioc|http://rdfs.org/sioc/ns#
+|foaf|http://xmlns.com/foaf/0.1/
+|pingback|http://purl.org/net/pingback/
+|as|http://www.w3.org/ns/activitystreams#
+|schema|https://schema.org/
+
+All RDF examples are written in Turtle syntax.
+
+
+## Discovery
+
+An inbox container must be discoverable via the `solid:inbox` property, and
+should be instances of `solid:Inbox`. Each notification can be discovered via
+`ldp:contains` relations. Notifications that are of LDP RDF Source should be
+instances of `solid:Notification`. Although neither `solid:Inbox` or
+`solid:Notification` is ultimately required to discover inboxes or
+notifications, it is good practice to instantiate resources for the purpose of
+data integrity and reuse. An example:
+
+```
+<https://example.org/profile> solid:inbox <https://example.net/inbox/> .
+<https://example.net/inbox/> a solid:Inbox .
+<https://example.net/inbox/> ldp:contains <https://example.net/inbox/abc123> .
+<https://example.net/inbox/abc123> a solid:Notification .
+```
+
+
+## Notification integrity
+
+Adhering to the integrity rules of an inbox notification e.g., for the purpose
+of a verification criteria or discovery is the responsibility of applications
+which submit and verify them. An inbox may be accompanied with constraint
+information e.g., by employing the [Shapes Constraint
+Language](https://www.w3.org/TR/shacl/)(SHACL) or [Shape
+Expressions](https://www.w3.org/2001/sw/wiki/ShEx)(ShEx).
+
+
+## Enrichment
+
+Although providing enriched (meta)data is voluntary, provenance level data like
+the date in which the request was submitted, who submitted, and its license, can
+be purposed towards the verification process as well as for displaying.
+
+
+## Authorization
+
+The owner of the Solid inbox typically sets its ACL rules and thereby for the
+notifications it contains. For example, notifications may be set for public read
+and append so that applications can use a FYN approach to discover and display
+the contents of a notification, as well as submit new notifications. Further,
+the ACL settings for the inbox may determine which agents can send
+notifications.
+
+
+## Notification types
+
+Here we exemplify some types of notifications:
+
+### Single statement
+
+A single triple statement is the simplest notification in terms of its data
```

Comment:
Hmm, not necessarily, but perhaps you can clarify. `solid:Notification` is a 'should' because single statement notifications is a reasonable approach. And, yes, in that case, they obviously can't use `solid:Notification`.